data-serving/scripts/aggregate/aggregate/app.py

```diff
@@ -354,40 +358,43 @@ def generate_total_json():
         "total_b1351": int(total_b1351),
     }
 
-    s3 = boto3.client("s3")
+    s3 = boto3.client("s3", endpoint_url=s3_endpoint)
     s3.put_object(
         ACL="public-read",
         Body=json.dumps(record),
-        Bucket="covid-19-aggregates",
+        Bucket=bucket,
         Key="total/latest.json",
     )
     s3.put_object(
         ACL="public-read",
         Body=json.dumps(record),
-        Bucket="covid-19-aggregates",
-        Key=f"total/{now}.json",
+        Bucket=bucket,
+        Key=f"total/{date}.json",
     )
 
 
-def lambda_handler(event, context):
-    """Generates aggregate data for Map and uploads to S3.
-
-    Parameters
-    ----------
-    event: dict, required
-        Input event JSON-as-dict specified by EventBridge.
-    context: object, required
-        Lambda Context runtime methods and attributes.
-        For more information, see:
-          https://docs.aws.amazon.com/lambda/latest/dg/python-context-object.html
-    """
+if __name__ == '__main__':
+    connection_string = os.environ.get("CONN")
+    line_list_url = os.environ.get("LINE_LIST_URL")
+    map_url = os.environ.get("MAP_URL")
+    s3_bucket = os.environ.get("S3_BUCKET")
+    # S3 endpoint is allowed to be None (i.e. connect to default S3 endpoint),
+    # it's also allowed to be not-None (to use localstack or another test double).
+    s3_endpoint = os.environ.get("S3_ENDPOINT", None)
+    print("Logging into MongoDB...")
+    client = pymongo.MongoClient(connection_string)
+    db = client.covid19
+    cases = db.cases
+    print("And we're in!")
+
+    date_string = datetime.datetime.now().strftime("%m-%d-%Y")
```

Comment:
I calculate this once in case the script ever runs over midnight so it uses a consistent date for one invocation.